VERSION

```diff
@@ -1 +1 @@
-1.11.1
+1.11.2
```

Comment:
Why are we incrementing the Conjur version in this PR?

Reply:
Returned it back to 1.11.1